File: single-language-analysis/dataset-files/java/TestSmell166.java
import org.junit.Test;
import static org.junit.Assert.*;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.firefox.FirefoxOptions;

public class TestSmell166 {

    @Test
    public void testBasicOptions() {
        FirefoxOptions options = new FirefoxOptions();
        FirefoxDriver driver = new FirefoxDriver(options);
        
        driver.quit();
    }
}

File: single-language-analysis/dataset-files/python/test_smell_166.py
# Original URL: https://github.com/fuatnku/seleniumhq.github.io/blob/76361bf094b4a044d1250079cb06f68dfb0a5027/examples/python/tests/browsers/test_firefox.py#L9-L13

def test_smell_166():
    options = webdriver.FirefoxOptions()
    driver = webdriver.Firefox(options=options)

    driver.quit()